test/config/FrontendAppConfigSpec.scala

```diff
@@ -43,7 +43,7 @@ class FrontendAppConfigSpec extends SpecBase {
     }
 
     "return the adjustments link if a CashStatement FileRole provided" in new Setup {
```

Comment:
Could you please replace CashStatement with CDSCashAccount?

Reply:
Replaced CashStatement with CDSCashAccount